requirements.txt

```diff
@@ -11,6 +11,7 @@ sphinx>=2.1.2,<=4.0.2
 breathe>=4.14.1
 # Jinja2 v3 breaks nbsphinx
 jinja2==2.11.3
+MarkupSafe==2.0.1
```

Comment:
this is needed because of the pin of jinja to version 2 above, newer versions of MarkupSafe are no longer compatible with it